backend/src/routes/matching.js

```diff
@@ -0,0 +1,65 @@
+const express = require('express')
+const router = express.Router()
+// const debug = require('debug')('app:students')
+const statuses = require('../helpers/statuses.js')
+const statusUpdates = require('../helpers/statusUpdates.js')
+const { NotFoundError } = require('objection')
+
+// TODO: Handle errors
+
+/* POST multiple students to screening endpoint */
+// on Postman: http://localhost:3001/students/1
+router.post('/', async (req, res) => {
```

Comment:
Looks like this whole implementation is not done yet - it's still the implementation from status update.